src/app/forms/buttons-page/buttons-page.component.html

```diff
@@ -1,14 +1,14 @@
 <h3 class="style-header">Buttons</h3>
 <ngx-section class="shadow" sectionTitle="Buttons">
   <button type="button" class="btn">Default</button>
-  <button type="button" class="btn btn-primary">Primary</button>
+  <button type="button" class="btn btn-primary btn-primary-gradient">Primary</button>
```

Comment:
Let's keep the `primary` examples in there, and just add the `primary-gradient` ones in addition to those.